pages/md-style-guide.md

```diff
@@ -1,191 +1,165 @@
+We use the [Google developer documentation style guide](https://developers.google.com/style) as our default guide. For a quick summary, see the [Google style guide highlights](https://developers.google.com/style/highlights). This document describes customizations to the Google developers style guide sepcific to the JSON Schema documentation. 
 
-```markdown
-<Star label="label"/>
-```
+The JSON Schema documentation uses American spelling and our preferred dictionary is the
+[American Heritage Dictionary](https://ahdictionary.com/). 
 
-### Example
+## Write for your audience
 
-<Star label="label"/>
+The content of our documentation is written mainly for develepers, therefore, information should be clear, concise, and accurate. If you're unsure about a certain detail, please contact the appropriate people to verify it. For more information about our users personas, see [Personas](https://github.com/json-schema-org/community/issues/336#personas). 
 
----
-
-## StarInline
-
-Adds an inline star with the label along with it. In place of `"label"` text can be added which needs to be shown.
-
-```markdown
- <StarInline label="label"/>
-```
-
-### Example
-
-<StarInline label="label"/>
-
----
-
-## Summary
-
-Summary and details tags both can be used together the text wrapped in the `summary` tag will be shown primarily, whereas the text shown in the `details` tag can be shown when its cliced on the summary text.
-
-```markdown
- <summary> message to show </summary>
-```
-
-### Example
-
-<summary> message to show </summary>
-
----
-
-## Details
+## Types of documents and information architecture
 
-Summary and details tags both can be used together the text wrapped in the `summary` tag will be shown primarily, whereas the text shown in the `details` tag can be shown when its cliced on the summary text.
+To better attend the needs of our readers, we create technical documentation according to the [diataxis framework](https://diataxis.fr/). Therefore, most of our documents can be classified as one of the following:
 
-```markdown
-<details> message to show </details>
-```
+- Reference
+- Tutorial
+- How-to guides
+- Explanation
 
-### Example
+Other types of documents we write are:
 
-<details> message to show </details>
+- Use cases
+- Case studies
+- Release notes
 
----
-
-## Bigquote
+We organize our documentation into the following content buckets:
 
-Useful to show the text wrapped in it, in a big quotation-like text.
+- **Introduction**: Several types of documents live in this bucket that introduce the JSON Schema initiative's purpose, how the JSON Schema is used in various industries and applications, and its key features and benefits.
+   
+- **Basics**: The documents of this content bucket help beginners get started with JSON Schema. 
+  
+- **Guides**: The guides contained in this bucket help users develop practical skills for validating schemas, building complex ones, and troubleshooting issues. 
```

Comment:
Should we add existing in the sentence? 

'for validating **existing** schemas, building complex ones, and troubleshooting issues.'